src/Sentry/Internal/Http/CachingTransport.cs

```diff
@@ -75,35 +75,35 @@ private async Task CachedTransportBackgroundTaskAsync()
                         File.Move(filePath, destinationPath);
                     }
                 }
+            }
+            catch (Exception e)
```

Comment:
Moved this here since below it would never be reached due to the `catch (Exception)` within the `try` block